config.go

```diff
@@ -123,6 +125,7 @@ type config struct {
 	TestNet3             bool          `long:"testnet" description:"Use the test network"`
 	RegressionTest       bool          `long:"regtest" description:"Use the regression test network"`
 	SimNet               bool          `long:"simnet" description:"Use the simulation test network"`
+	Checkpoints          []string      `long:"checkpoint" description:"Add a custom checkpoint. Format: '<height>:<hash>'"`
```

Comment:
I wonder if the option should be `--addcheckpoint` so it is more consistent with `addpeer` and also makes it a little more obvious that it is adding one or more checkpoints as opposed to replacing them?